packages/base-types/src/trace/v3/text.ts

```diff
@@ -0,0 +1,11 @@
+import { Markup } from '@base-types/markup';
+
+import { BaseTraceFrame, TraceType } from '../../node/utils';
+
+interface StepData {
+  content: Markup;
```

Comment:
The data payload is subject to change due to a proposal by Tyler H to simply return Markdown rather than Markup. See slack thread [here](https://voiceflowhq.slack.com/archives/CGFFRC588/p1691469086204979).